### PR TITLE
tweaks to onboard sketch

### PR DESCRIPTION
**Pull request by ryanatkn**
Makes a lot of small adjustments to the consentful onboarding sketch. I think it's about ready to call "finished" if we can believe that. Deployed via `gro deploy --branch tweaks`

# 🐿️

### CODE REVIEW

**Review by greatbacon — APPROVED** [review]
🎉 Should we make a twitter post about this now?

File: src/lib/sketch/onboard/End.svelte [hunk 1]
```diff
@@ -9,19 +9,18 @@
...
-		<a href="https://github.com/feltcoop/felt">on GitHub</a>
+		questions or comments? open
+		<a href="https://github.com/feltcoop/felt">a GitHub issue</a>
 		or email us at <a href="mailto:team@felt.social">team@felt.social</a>
 	</blockquote>
 	<Credits />
 	<blockquote>
 		<p>
-			if you'd like to explore freely, press the <code>Backtick</code> key or click this button:
+			to explore freely, press the <code>Backtick</code> key or click this button:
```
Comment (greatbacon) [inline src/lib/sketch/onboard/End.svelte:21]: One note, should we replace `Backtick` with the literal <code>`</code>?

**Comment by ryanatkn** [comment]
Left's draft something!

File: src/lib/sketch/onboard/End.svelte (hunk 1 above)
Comment (ryanatkn) [reply src/lib/sketch/onboard/End.svelte]: Changed to this, what do you think?

```svelte
to explore freely, press <code>`</code> aka <code>Backtick</code> or click this button:
```

Comment (ryanatkn) [reply src/lib/sketch/onboard/End.svelte]: Looking at this I think I got a little overzealous editing things down, I undid this change:

```svelte
<p>We appreciate your time. We hope you learned something & had some fun in the process {grin}</p>
```

I reverted partially so it's now:

```svelte
<p>
	We appreciate your time going through this sketch. We hope you learned something & had some fun
	in the process {grin}
</p>
```

original:

```svelte
	<p>We appreciate you taking the time to go through this sketch.</p>
	<p>We hope you learned something & had some fun in the process {grin}</p>
```

Any thoughts?

Comment (greatbacon) [reply src/lib/sketch/onboard/End.svelte]: I like tighter sentences, so I'm good with the current iteration. Thanks for the clarification on the ` too!

**Pull request merged**